releasenotes/releasenotes_V0.7.2.md

```diff
@@ -0,0 +1,93 @@
+# Release Notes 0.7.2
+
+Keptn 0.7.2 improves the capabilities of the 0.7.0 release by adding ... 
+
+**The key announcements of Keptn 0.7.2**:
+
+:rocket: *Improved UX for Quality Gates*: This release introduces an `/evaluation` endpoint that brings the same user-experience to the API as used from the Keptn CLI. Hence, easily trigger a quality gate evaluation based on a small set of parameters, like: `start`, `end`, and `timeframe`. 
```

Comment:
Should we add here that the provided project, service and stage are also evaluated?